engineering_tools/opendrive_lanelet2_converter/README.md

```diff
@@ -0,0 +1,25 @@
+# opendrive2lanlet2
+
+opendrive2lanlet2 is a Python script for converting opendrive maps to lanelet2 map readable by autoware.
+
+## Installation
+
+as a dependency need to install opendrive2lanelet library with a GNU General Public License v3.0
+
+```bash
+pip install opendrive2lanelet
+```
+
+## Usage
+
+```bash
+python3.7 map_convertor.py -i input.xodr -o output.osm
+```
+
+## Description
+
+Node class representing node in lanelet2
+Way class representing way in lanelet2
```

Comment:
See comment on script about 'way'

I think generally a little bit more description on how this script works would be beneficial, such as 
3rd party converter outputs to common road xml file which is then converted into lanelet2 osm.

Reply:
there was no need to create a common road xml format, just needed to use the lanelets list to create a lanelet2 xml file.

Reply:
You might not be writing it to a file but you are converting it first to common road then osm. I am talking about the internal functioning of the script which is important for any future developer to understand.

Reply:
can you point me to which part of script you talking about ?